### PR TITLE
.Net core 3.1

### PR DESCRIPTION
**Pull request by ada3m0n666**
just a quick update to get nswag work on .net core 3.1.

### CODE REVIEW

File: src/NSwag.MSBuild/NSwag.MSBuild.nuspec [hunk 1]
```diff
@@ -27,11 +27,12 @@
     <file src="..\NSwag.Console.x86\bin\Release\net461\NSwag.x86.exe" target="tools/Win" />
     <file src="..\NSwag.Console.x86\bin\Release\net461\NSwag.x86.exe.config" target="tools/Win" />
 
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.0\Publish\**" target="tools/NetCore10" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.1\Publish\**" target="tools/NetCore11" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.0\Publish\**" target="tools/NetCore20" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.1\Publish\**" target="tools/NetCore21" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.2\Publish\**" target="tools/NetCore22" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.0\Publish\**" target="tools/NetCore30" />
+    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.0\**" target="tools/NetCore10" />
```
Comment (RicoSuter) [inline src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: Why no /Publish?

Comment (ada3m0n666) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: oh yes sorry. When i  try to generate manually nuget package from Nswag.MsBuild it tells me nothing present in /Publish. To get my package quickly i remove /publish. Did not have time to investigate on this sorry.

Comment (ada3m0n666) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: to get build.bat works

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: strange. But the build is huge and requires some sdks etc

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: strange. But the build is huge and requires some sdks etc

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: strange. But the build is huge and requires some sdks etc

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: strange. But the build is huge and requires some sdks etc

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: Will check tomorrow

Comment (ada3m0n666) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: i can cancel my changes on /publish if you want.

Comment (RicoSuter) [reply src/NSwag.MSBuild/NSwag.MSBuild.nuspec]: Not sure but i think /publish is needed in order that this works correctly.

**Comment by RicoSuter** [comment]
Please do not upgrade to v13.1.7, I'll to that in another PR.

**Comment by bhugot** [comment]
@RicoSuter why the solution doesn't use global.json file? would ease the build

File: src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <_NSwagCommand>$(NSwagExe)</_NSwagCommand>
     <_NSwagCommand
-        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core21)/dotnet-nswag.dll</_NSwagCommand>
+        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core31)/dotnet-nswag.dll</_NSwagCommand>
```
Comment (RicoSuter) [inline src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets]: I think we need to keep this on v3.0? /cc @dougbu

Comment (dougbu) [reply src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets]: Actually, this should stay at 2.1 to allow the tool to run in as many supported environments as possible.

Comment (RicoSuter) [reply src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets]: Ok thanks for the reply, will change it back to 2.1

**Comment by RicoSuter** [comment]
v13.2.0

**Comment by RicoSuter** [comment]
Please report if its working for you.

**Comment by paulovila** [comment]
Maybe I  don't know how to configure it, the nuget package for nswag.msbuild 13.2.0 doesn't contain the target folder for NetCore3.1, is that still ok? In the  msbuild task (csproj), what version should it be used? Maybe I don't know what runtime should I use in the nswag configuration file (runtime:NetCore31)

**Comment by RicoSuter** [comment]
Then something must be wrong

**Comment by RicoSuter** [comment]
This looks good
https://github.com/RicoSuter/NSwag/blob/master/src/NSwag.MSBuild/NSwag.MSBuild.nuspec#L36

**Comment by paulovila** [comment]
In my nuget folder, the nswag.msbuild folder doesn't contain  the NetCore3.1 subfolder..., 
If you download the  nuget package directly and expand the  zip file, do you see the NetCore3.1 folder?

**Comment by JamesFenton** [comment]
Looks fine to me (downloaded from nuget.org)

image

image

**Comment by paulovila** [comment]
Great, I've tried in another pc, and I can see tools/NetCore31 being generated, I'll clear the nuget cache,  

thanks!

**Pull request merged**